variables.tf

```diff
@@ -138,3 +138,9 @@ variable "bucket_id" {
   default     = ""
   description = "Used to find the parent resource to bind the IAM policy to"
 }
+
+
+variable "bucket_iam_members" {
+  type = list(string)
```

Comment:
Please provide description for variable, and also provide default value (if required.).

Reply:
Done